AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -666,10 +666,13 @@ - (ASLayout *)transitionLayoutWithAnimation:(BOOL)animated
 
 - (ASLayout *)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize animated:(BOOL)animated
 {
+  BOOL disableImplicitHierarchyManagement = self.usesImplicitHierarchyManagement == NO;
```

Comment:
Ok this is a bit of a nit, but would it be easier to just capture the original value of usesImplicitHierarchyManagement and restore it unconditionally?